src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -268,6 +284,55 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
     cleanupTestData(saveModeUnit)
   }
 
+  it should "check for null ids on time series update" taggedAs WriteTest in {
+    val df = spark.read
+      .format("com.cognite.spark.datasource")
+      .option("apiKey", writeApiKey.apiKey)
+      .option("type", "timeseries")
+      .load()
+    df.createOrReplaceTempView("sourceTimeSeries")
+
+    val initialDescription = "'post testing'"
```

Comment:
Looks like we haven't been consistent in this file wrt. whether or not to add the single quotes in the value definition, or add it where the value is referenced. We've been consistent in the other test files though, so I suggest removing the single quotes from the value definition here and adding them to the string where they're referenced.